sormas-backend/src/main/java/de/symeda/sormas/backend/person/PersonFacadeEjb.java

```diff
@@ -312,6 +316,16 @@ public JournalPersonDto getPersonForJournal(String uuid) {
 		}
 	}
 
+	public String getInternationalPhoneNumber(String phoneNumber) {
```

Comment:
Is this method used anywhere?

Reply:
No, I will remove it

Reply:
You did not remove it. If you want to keep it, please comment accordingly in this thread and explain why. 
I find the method misleading, because it does not always return an internationalized phone number. So I would suggest to either change the method behaviour, write a javadoc, or remove the method.